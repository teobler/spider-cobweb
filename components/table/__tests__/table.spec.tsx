import * as React from 'react';
import { Table, TableBody, TableCell, TableFooter, TableHead, TableRow } from '../../index';
import renderer from 'react-test-renderer';

describe('<Table />', () => {
  it('should render a empty table', () => {
    const menuJson = renderer
      .create(
        <Table>
          <TableHead>
            <TableRow>
              <TableCell />
            </TableRow>
          </TableHead>
          <TableBody>
            <TableRow>
              <TableCell />
            </TableRow>
          </TableBody>
          <TableFooter>
            <TableRow>
              <TableCell />
            </TableRow>
          </TableFooter>
        </Table>,
      )
      .toJSON();
    expect(menuJson).toMatchSnapshot();
  });
});
