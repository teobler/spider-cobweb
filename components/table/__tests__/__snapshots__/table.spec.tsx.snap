// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> should render a empty table 1`] = `
<table
  className="spider-table"
>
  <thead
    className="spider-table-head"
  >
    <tr
      className="spider-table-row"
    >
      <td
        className="spider-table-cell"
      />
    </tr>
  </thead>
  <tbody
    className="spider-table-head"
  >
    <tr
      className="spider-table-row"
    >
      <td
        className="spider-table-cell"
      />
    </tr>
  </tbody>
  <tfoot
    className="spider-table-footer"
  >
    <tr
      className="spider-table-row"
    >
      <td
        className="spider-table-cell"
      />
    </tr>
  </tfoot>
</table>
`;
